imports/ui/components/reviews/reviews_update.html

```diff
@@ -0,0 +1,27 @@
+<template name='reviewsUpdate'>
+  {{#Layout template=materializeHeader}}
+    {{collection.title}}
+  {{/Layout}}
+  {{#Layout template=materializeContent}}
+    {{#if Template.subscriptionsReady}}
+      {{#autoForm collection=collection doc=item id="updateReviewForm" type="update"}}
+        {{> afQuickField name="artist"}}
+        {{> afQuickField name="releaseName"}}
+        {{> afQuickField name="year"}}
+        {{> afQuickField name="label"}}
+        {{> afQuickField name="rating"}}
+        {{> afQuickField name="body"}}
+        {{#if isMod}}
+          {{> afQuickField name="approved"}}
+        {{/if}}
+      {{/autoForm}}
+    {{/if}}
+  {{/Layout}}
+  {{# Layout template=materializeButtons }}
+      <a href="{{ collection.indexPath }}" class="waves-effect waves-light btn {{ orion_autoformLoading }}">{{ i18n 'global.cancel' }}</a>
+      {{# if item.canShowRemove }}
+        <a class="waves-effect waves-light btn red {{ orion_autoformLoading }}" href="{{ collection.deletePath item._id }}">{{ i18n 'global.delete' }}</a>
+      {{/ if }}
+      <a class="waves-effect waves-light btn light-blue accent-4 save-btn {{ orion_autoformLoading }}" href="#">{{ i18n 'global.save' }}</a>
+  {{/ Layout }}
+</template>
```

Comment:
Oh, I see! I wasn't quite able to piece that together from the code alone. That's very clever. And thanks for the links; it could be helpful to have those in here somewhere.